templates/es-data.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: {{ if .Values.common.stateful.enabled }}apps/v1beta1{{ else }}extensions/v1beta1{{ end }}
+apiVersion: {{ if .Values.common.stateful.enabled }}apps/v1beta2{{ else }}extensions/v1beta1{{ end }}
```

Comment:
I believe we should add a guard here, so pre1.8 versions don't break. Helm provides information about the cluster so you can choose the appropriate API version based on it.

Reply:
some features are not available pre-1.8 like GID settings (very important to be able to use official ES containers)